.gitignore

```diff
@@ -0,0 +1,30 @@
+# macOS system files
```

Comment:
Is a `.gitignore` needed here?

Reply:
It was catching some DS_store files from my mac, that is why I included it.

Reply:
Yes

Reply:
The file is already on the root folder, do you want to move it?